.gitignore

```diff
@@ -51,4 +51,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
-vendor/
\ No newline at end of file
+vendor/
+/.idea/kotlinc.xml
```

Comment:
nit: maybe bundle up with the others starting on line 21 to organize, or just exclude `/.idea/*` at this point?

edit: i see right below at least one .idea file for detekt that would make the * a bad idea probably.